packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -110,6 +117,23 @@ const createEndAdditionalResultsStack = (count, interpretationsArr, hasAccessory
   ].some(r => interpretationsArr.indexOf(r) >= 0)) {
     additionalResultInterpretationIndicator = 'high';
   }
+  let criticality;
+  if ([
+    'critical',
+    'critical-high',
+    'critical-low',
+    'positive',
+  ].some(r => interpretationsArr.indexOf(r) >= 0)) {
+    criticality = 'critical';
+  } else if ([
+    'abnormal',
+    'high',
+    'low',
+  ].some(r => interpretationsArr.indexOf(r) >= 0)) {
+    criticality = 'out of range';
+  } else {
+    criticality = 'normal';
```

Comment:
So when all results are normal what will be read out is "A total of {count} results are available with at least 1 normal result." 

It could just be me but that sounds a little weird. When the result is critical or out of range that makes sense to call out that there is 'at least' one result of those types since it's important additional information to know. But when they are all normal saying 'at least 1 normal result' could potentially sound confusing, like not all of them are normal.

What do you think? Maybe we could add some logic that when all are normal we either don't add the 'at least 1 normal result' or instead say 'all results normal'. I know this string isn't completely approved yet but I did want to bring this up.

Reply:
Alright looks like Brett and Clayton got back to us on the Jira - they said for now the best choice will be to not add the 'at least 1 normal result' to the end of the string when all results are normal. So just "A total of {count} results are available".